### PR TITLE
Setup project dependencies

### LINKED ISSUE #6: Setup project dependencies
We need to set up the project dependencies.

- Reference: https://github.com/parafoxia/analytix/tree/main/requirements

### PR DESCRIPTION
**Pull request by Jonxslays**
This PR adds the requirements dir like how analytix does it. I tried to take only the deps we need.

Let me know if I'm missing anything.

**Yes thats pyright strict you see there.**

Closes #6

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+aiofiles>=0.5,<=23.2.1
+aiohttp>=3.8.1,<4
```
Comment (parafoxia) [inline requirements/base.txt:2]: We'll need click here as well for the CLI stuff.

**Review by parafoxia — CHANGES_REQUESTED** [review]
LGTM, though the setup.py needs to be updated with these requirements.

File: requirements/base.txt (hunk 1 above)
Comment (Jonxslays) [reply requirements/base.txt:2]: oh yeah! will add.

**Comment by Jonxslays** [comment]
> LGTM, though the setup.py needs to be updated with these requirements.

see, this is why we're making severn.

**Comment by parafoxia** [comment]
> > LGTM, though the setup.py needs to be updated with these requirements.
> 
> see, this is why we're making severn.

Literally yeah! :ainsley: (GitHub custom emojis when)

**Review by parafoxia — APPROVED** [review]

**Pull request merged**